page/pindad-in-news.php

```diff
@@ -0,0 +1,109 @@
+<link rel="stylesheet" href="css/template-1.css">
+<div class="full text-center">
+    <span class="garis"></span>
+    <h1 class="display-5 text-light"><dt><b>PINDAD DALAM BERITA</b></dt></h1>
+</div>
+
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-8">
+
+            <?php
+                $select = mysqli_query($connection, "select * from berita where kategori='pindad-in-news'");
```

Comment:
Codacy found an issue: [The use of function mysqli_query() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)